test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
- [ ] Please make sure this file `test.md` is properly ignored from the git before making a git push 🙏